src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -1694,6 +1685,26 @@ module Base = struct
             with_label __LOC__
               (fun () -> Field.Assert.equal (implied_root account incl))
               (Ledger_hash.var_to_hash_packed root)
+
+          let check_account public_key token_id
+              (({ data = account; _ }, _) : Account.t * _) =
+            let ( ! ) = run_checked in
```

Comment:
Not sure this is saving characters, overall, it's only used twice. It may make the code less readable, since it can confused with dereferencing.